src/main/cpp/t06_sum.cpp

```diff
@@ -22,7 +22,23 @@
 
 using namespace std;
 
+int  sum()
+{
+	int n;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'n' is not assigned a value.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951807)